### ISSUE TITLE
Activating/Deactivating a Font file

### ISSUE THREAD

**Issue by YashAgarwal** [sugarlabs/edit-fonts-activity#43]
I can use the following to install a .ttf or .otf file in sugar but I require root access for it

``` bash
mv path/to/font/file /usr/share/fonts
fc-cache -f -v 
```

I can use `subprocess` module to run the commands through my activity but how do I provide it root access or is there another way to do this?

**Comment by samdroid-apps** [comment]
Can you use the `~/.fonts` directory?  That is how I usually install 
fonts on my computer :)

**Comment by YashAgarwal** [comment]
@samdroid-apps  yeah I can, but that folder doesn't exist on my sugar environment, all the fonts were stored in /usr/share/fonts
won't my activity need root for writing to that directory to?

**Comment by samdroid-apps** [comment]
No, just create a `~/.fonts` directory.

**Comment by walterbender** [comment]
You don't need root for -/fonts

**Comment by YashAgarwal** [comment]
oh, thanks @walterbender

**Comment by YashAgarwal** [comment]
if the font file is in `/user/share/fonts` then I won't be able to **deactivate** the font file by moving the file out of the folder and running `fc-cache -f -v` or is there any other way of deactivating font without having root privileges.

**Comment by davelab6** [comment]
@YashAgarwal you should be able to get root with `gksudo` (http://askubuntu.com/questions/51572/why-should-i-use-gksudo-for-gtk-apps-instead-of-sudo) 

However, one of the things Eli and I discussed yesterday in the youtube video is that the editor could by default only deal with UFOs; that means you could avoid dealing with `/usr/share/fonts` entirely

**Comment by davelab6** [comment]
I believe this is now ready for testing in the next .xo release

**Comment by davelab6** [comment]
On Ubuntu 14.04, the fonts folder path is `~/.local/share/fonts` so we'll need to account for that too.

**Comment by davelab6** [comment]
Next action is to uncomment this code in PR #74
